TeamCode/src/main/java/org/firstinspires/ftc/teamcode/core/lib/autonomousControl/TargetVelocityData.java

```diff
@@ -0,0 +1,44 @@
+package org.firstinspires.ftc.teamcode.core.lib.autonomousControl;
+//this is done
+public class TargetVelocityData {
+    double XVel;
+    double YVel;
+    double XV_YV_alpha;
```

Comment:
review variables to always start with lower case + you should use _ only when creating a const
the const patter is CONST_NAME

Reply:
class removed as per another review resolution. But I agree, I now see how inconsistant i was with my naming convections while developing this code